### ISSUE TITLE
Show the searchbar on click of zet icon

### ISSUE THREAD

**Issue by kaushal087** [nandwana92/zet#6]
## Summary
Show the same searchbar on click of zet icon which opens on Command/Control + Shift +Space.

<img width="414" alt="Screenshot 2020-05-07 at 4 54 25 PM" src="https://user-images.githubusercontent.com/4817933/81289214-a2859380-9083-11ea-9209-5c0f73350521.png">

**Comment by nandwana92** [comment]
Yes, this makes sense. A user would intuitively think clicking on it should open the Popup. I will put this in the next release.
